extensions/amazon-lambda/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -7,4 +7,5 @@ metadata:
   - "amazon"
   categories:
   - "cloud"
+  guide: "https://quarkus.io/guides/amazon-lambda"
```

Comment:
It seems the guide link was missing here. To be confirmed.

Reply:
Looks good.